LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -68,13 +68,19 @@ public override void Configure(IFunctionsHostBuilder builder)
                 .AddSingleton<IFunctionBundlerExecutionItem, ADRExecutionItem>()
                 .AddSingleton<IFunctionBundlerExecutionItem, PreferredGatewayExecutionItem>()
                 .AddSingleton<LoRaDevAddrCache>();
+
+            if (!string.IsNullOrEmpty(configHandler.AppInsightsInstrumentation))
+            {
+                _ = builder.Services.AddApplicationInsightsTelemetry(configHandler.AppInsightsInstrumentation);
```

Comment:
are you sure you need the key here? I think there is an overload that does not take a key and will try to use the default configuration? Did you try that?

Reply:
I tried now, it seems to be working actually 👍 will remove the code around getting the key.